src/components/ui/colorPickerInput/index.tsx

```diff
@@ -45,28 +35,21 @@ class ColorPickerInput extends Component<IProps> {
       disabled = false,
       className = null,
       type = 'text',
-      error = '',
       onChange = noop,
     } = this.props;
 
     return (
       <div
         className={classnames({
           'franz-form__field': true,
-          'has-error': error,
           [`${className}`]: className,
         })}
         ref={this.inputElement}
       >
         <SliderPicker
-          color={value}
-          onChange={this.onChange.bind(this)}
-          id={`${id}-SliderPicker`}
-          type={type}
```

Comment:
Since I installed `@types/react-color` there were a handful of TS issues -> I removed all the props which `react-color` doesnt even accept.